paragraph/scheduling/logger.cc

```diff
@@ -56,26 +56,27 @@ absl::Status Logger::LogInstruction(
 
 Logger::Logger(const std::string& filename)
     : filename_(filename) {
-  std::fstream ostream_;
+  std::fstream log_stream_;
```

Comment:
Why are you redeclaring this here? This "log_stream_" will not be the same as "this->log_stream_". The names are aliased and by the way this stream doesn't get used :)